### PR TITLE
Update #499 - adds backend for star comments in user profile (backend)

### PR DESCRIPTION
**Pull request by kondanna**
adds additional fields for 
query { userInfo(<username>) { lessonStatus { starsReceived { <additional fields> } } } }
in order to show student initials, student username, lesson name, and comment

### CODE REVIEW

File: graphql/typeDefs.ts [hunk 1]
```diff
@@ -170,8 +170,12 @@ export default gql`
   type Star {
     id: String!
     studentId: Int
+    studentUsername: String
+    studentName: String
     mentorId: Int
     lessonId: Int
+    lessonTitle: String
+    lessonDifficulty: Int
```
Comment (Ulisseus) [inline graphql/typeDefs.ts:178]: `lessonDifficulty` implies that some lessons are easier than other. Is this just `lessonId`? (eg. `Foundations` lesson has id `5`, Variables lesson has id `2` and etc?)

Comment (kondanna) [reply graphql/typeDefs.ts:178]: ah, it's actually the 'order' of the lesson. the lesson IDs are kinda all over the place.

File: helpers/controllers/userInfoController.ts [hunk 2]
```diff
@@ -35,11 +35,26 @@ export const userInfo = async (_parent: void, args: Username) => {
...
     map[star.lessonId] = map[star.lessonId] || []
+    const {
+      student: { username, name },
+      lesson: { order, title }
+    } = star
+    star.dataValues = {
+      ...star.dataValues,
+      studentUsername: username,
+      studentName: name,
+      lessonDifficulty: order,
```
Comment (songz) [inline helpers/controllers/userInfoController.ts:55]: I think `lessonOrder`         is more fitting

**Comment by kondanna** [comment]
created new branch / pull request after pulling from upstream master

**Pull request closed without merging**